pymodm/base/fields.py

```diff
@@ -98,10 +98,21 @@ def __set__(self, inst, value):
 
     def __delete__(self, inst):
         inst._data.pop(self.attname, None)
+        inst._defaults.pop(self.attname, None)
```

Comment:
if hasattr(inst, '_defaults'):?

Or we can just define _defaults in MongoModelBase.